tools/obj_rec_ransac_result.cpp

```diff
@@ -347,10 +347,10 @@ loadScene (const char* file_name, PointCloud<PointXYZ>& non_plane_points, PointC
 
   // Make sure that the ids are sorted
   sort (inliers->indices.begin (), inliers->indices.end ());
-  std::size_t j = 0;
-  for ( std::size_t i = 0, id = 0 ; i < inliers->indices.size () ; )
+  std::size_t j = 0, i = 0;
+  for ( pcl::index_t id = 0 ; i < inliers->indices.size () ; )
```

Comment:
We can move the ++id from L364 to here. This is a tool that could do with some minor refactoring down the road.

i and j are also indices into pointcouds, so they could also be index_t

Reply:
Done, and agreed on the refactoring (other tools probably as well)